src/coredump/coredump.c

```diff
@@ -1173,6 +1183,7 @@ static int process_kernel(int argc, char* argv[]) {
         Context context = {};
         struct iovec_wrapper *iovw;
         int r;
+        _cleanup_free_ char *process_name = NULL;
```

Comment:
Move this declaration inside the else {} block - but above the initialization, not on the same line

Reply:
Addressed the comment. Please check once.